test/e2e/framework.go

```diff
@@ -213,9 +217,16 @@ type TestOptions struct {
 	// the home directory of the control-plane Node. Note it doesn't affect the tests that redeploy Antrea themselves.
 	deployAntrea bool
 
-	externalServerIPs string
-	vlanSubnets       string
-	vlanID            int
+	externalAgnhostIPs string
+	vlanSubnets        string
+	vlanID             int
+
+	externalFRRIPs string
+	// FRR cannot currently be configured remotely over networking. As a result, the e2e tests for BGPPolicy can only
+	// be run in a Kind cluster, where the FRR container can be configured using Docker exec with the container ID.
+	// TODO: Introduce a BGP router implementation that can be configured remotely  over networking to replace FRR.
+	//   This would allow the e2e tests for BGPPolicy to be run in environments other than just a Kind cluster.
```

Comment:
```suggestion
	// TODO: Introduce a BGP router implementation that can be configured remotely over networking to replace FRR.
	// This would allow the e2e tests for BGPPolicy to be run in environments other than just a Kind cluster.
```